components/gitpod-db-go/conn.go

```diff
@@ -0,0 +1,37 @@
+package db
+
+import (
+	"fmt"
+	driver_mysql "github.com/go-sql-driver/mysql"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	"time"
+)
+
+type ConnectionParams struct {
+	User     string
+	Password string
+	Host     string
+	Database string
+}
+
+func Connect(p ConnectionParams) (*gorm.DB, error) {
+	loc, _ := time.LoadLocation("UTC")
+	cfg := driver_mysql.Config{
```

Comment:
note: we need to make sure we disable MySQLs "convert timestamps based on local connection timezone" feature, or it will auto-magically shift timezones "for us".

Reply:
It does that on line 26 in this file